src/clj/rems/db/users.clj

```diff
@@ -47,7 +47,8 @@
 
   You should use get-user instead."
```

Comment:
This line could be updated now that there are direct users of get-user-attributes to better reflect who should use get-user.